content/guides/vercel-sdk-migration.md

```diff
@@ -0,0 +1,233 @@
+---
+title: Migrate from Vercel Postgres SDK to the Neon serverless driver
+subtitle: Learn how to smoothly transition your Next.js application from the Vercel Postgres SDK to the Neon serverless driver
+author: dhanush-reddy
+enableTableOfContents: true
+createdAt: '2024-10-28T00:00:00.000Z'
+updatedAt: '2024-10-28T00:00:00.000Z'
+---
+
+With Vercel Postgres transitioning to Neon's native integration in the [Vercel Marketplace](https://vercel.com/blog/introducing-the-vercel-marketplace), now is the perfect time to migrate from the Vercel Postgres SDK (@vercel/postgres) to the Neon serverless driver. This guide will walk you through the migration process, helping you leverage the full power of Neon's serverless PostgreSQL capabilities.
+
+## Why migrate?
+
+Switching to the Neon serverless driver offers several advantages over the Vercel Postgres SDK:
+
+- **Direct database access**: Connect directly to your database without added abstraction layers.
+- **Enhanced performance**: Benefit from Neon's optimized database driver.
+- **Greater flexibility**: Choose HTTP for single queries or WebSockets for transactions.
+- **Improved maintainability**: Rely on Neon's actively maintained, native database driver.
+
+## Prerequisites
+
+To begin, you’ll need:
+
+- An existing Next.js application using the Vercel Postgres SDK
+- A [Neon account](https://neon.tech/docs/get-started-with-neon/signing-up) (your Vercel Postgres database will automatically migrate to Neon)
+- Basic knowledge of PostgreSQL and Next.js
```

Comment:
I think both of the knowledge items can be skipped?

Reply:
removed